src/index.scss

```diff
@@ -49,7 +49,7 @@ body {
     display: flex;
     width: 100%;
     flex-direction: column;
-    overflow: hidden;
+    overflow: scroll;
```

Comment:
I'm not sure it even needs `scroll`.  Think the `overflow` property could be removed altogether.

Reply:
Good point. Thanks. I'll try that and see...